specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -93,6 +93,10 @@ export function IntegratedRecordSelector({
 
   const isAttachmentTable = tables.Collection.name.includes('Attachment');
 
+  const isInSearchDialog = React.useContext(SearchDialogContext);
+  const isAttachmentPlugin = viewName === 'ObjectAttachment';
+  const isAttachmentInSearch = isInSearchDialog && isAttachmentPlugin;
```

Comment:
while #4226 only mentioned attachments, I belive this restriction should be applied to all records.
in fact, SubViews should probably not even be rendered on the search forms. @grantfitzsimmons what do you think?

the way sp6 solved this, is that it allows entering -to-many fields on the search form (like addresses.city) as if it is a -to-one for the purposes of searching (this can be seen on the default search form). I did not implement the same in sp7 as that is ugly (a special edge case), and makes validating forms harder - instead query builder should be encouraged for more complicated searches

Reply:
DECISION: 
- subview not displayed on search form

Reply:
I've thought through this more, I agree.